src/bun.js/webcore/response.classes.ts

```diff
@@ -153,7 +153,7 @@ export default [
       // This is *not* spec-compliant.
       name: {
         getter: "getName",
-        cache: true,
```

Comment:
we can still cache it right?

Reply:
that way it only creates the name once

Reply:
fixed: https://github.com/oven-sh/bun/pull/11815/commits/72453c87004fd9942fe40c0be08c9cff7adf2a5f